spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/data/redis/RedisClientPoolingException.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2012-2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.boot.autoconfigure.data.redis;
+
+import org.springframework.boot.autoconfigure.data.redis.RedisProperties.ClientType;
+import org.springframework.util.StringUtils;
+
+/**
+ * Exception thrown when a Redis client pooling failure.
+ *
+ * @author Weix Sun
+ */
+class RedisClientPoolingException extends RuntimeException {
```

Comment:
Thanks but I don't think we need this for now.

Reply:
Got it.